_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -0,0 +1,249 @@
+There are many situations in which an Identity is interested in an [IdentityAttribute]({% link _docs_integrate/data-model-overview.md %}#identityattribute) or a [RelationshipAttribute]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) of another Identity, for example:
+
+- You are interested in the [BirthDate]({% link _docs_integrate/attribute-values.md %}#birthdate) of another Identity so that you can wish this Identity a happy birthday every year.
+- You must know the [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress) of another Identity in order to be able to send an email to this Identity.
+
+In this guide, we explain how a Connector, hereinafter referred to as the Sender, can read an Attribute of another Connector, the so-called Recipient. Since understanding this reading process requires knowledge about [Requests]({% link _docs_integrate/data-model-overview.md %}#request) and how to use them in general, you should take a look at our Request and Response introduction before continuing reading this guide.
```

Comment:
For reasons of consistency, I'll leave "Sender" and "Recipient" capitalized for now, because that's how it's done in currently published documentation at the moment, e.g. https://enmeshed.eu/integrate/requests-over-messages.

Also, as in the currently published documentation (e.g. https://enmeshed.eu/integrate/requests-over-templates), I have now added a comment at the beginning that we are describing the reading process between two connectors for simplicity, but the general reading process between a connector and an app user would be the same.